<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralWidget</class>
 <widget class="QWidget" name="GeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="leftBar" native="true">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="farmingVariablesWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="farmingVariablesLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Farming Variables</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QWidget" name="farmingTechniqueWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="farmingTechniqueLabel">
              <property name="text">
               <string>Farming Technique:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="farmingTechniqueComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QWidget" name="unitOfMeasurementWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="unitOfMeasurementLabel">
              <property name="text">
               <string>Unit of Measurement:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="unitOfMeasurementComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QWidget" name="fuelTypeWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QLabel" name="fuelTypeLabel">
              <property name="text">
               <string>Fuel Type:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QComboBox" name="fuelTypeComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QWidget" name="amountOfFuelUsedWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QLabel" name="amountOfFuelUsedLabel">
              <property name="text">
               <string>Amount of fuel used:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignRight">
             <widget class="QDoubleSpinBox" name="amountOfFuelUsedDoubleSpinBox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QWidget" name="calculateContainerWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="calculateContainerCheckBox">
           <property name="text">
            <string>Local Temperatures</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="calculateContainerPushButton">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignBottom">
       <widget class="QWidget" name="moreContainerWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignBottom">
          <widget class="QWidget" name="moreWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="editDatabasePushButton">
              <property name="text">
               <string>Edit Database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="importPushButton">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportPushButton">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settingsPushButton">
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainBody" native="true">
     <property name="font">
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="mainBodyPlaceholderLabel">
        <property name="font">
         <font>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>main body</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignBottom">
       <widget class="QLabel" name="madeByCreditsLabel">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Made by Luis Rosario and Cameron Morgan</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightBar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QWidget" name="recentTransactionsHeaderWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QLabel" name="recentTransactionsHeaderLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recent Transactions</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="sqlTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>900</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignBottom">
       <widget class="QWidget" name="progressContainerWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>39</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignmentFlag::AlignVCenter">
          <widget class="QLabel" name="progressLabel">
           <property name="text">
            <string>nothing is happening</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignBottom">
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
